SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf

```diff
@@ -135,6 +123,7 @@
 	<owl:Class rdf:about="&fibo-sec-dbt-ab-pbs;StructuredFinanceWithoutCapitalProtection">
 		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ab-pbs;StructuredFinanceInstrument"/>
 		<rdfs:label xml:lang="en">structured finance without capital protection</rdfs:label>
+		<owl:disjointWith rdf:resource="&fibo-sec-dbt-ab-pbs;PrincipalProtectedNote"/>
 		<skos:definition xml:lang="en">structured finance that is a short-term note linked to an underlying asset that offers a steady stream of income</skos:definition>
```

Comment:
definition seems adrift from the class name which is about no protection. 
Also "Without Capital Protection" and "Principal Protected" are inconsistent.
 Why does it have to be "short term"?

Reply:
@rivettp - that is the definition from ISO, which we can revisit if we have a better source, but I changed the name to be consistent with principal-protected and added the synonym of capital protected

Reply:
If we do leave in "short term" we should reflect in some sort of restriction. And consider which other instruments are invariably "short term" including bonds. Maybe define the mixin class "short term debt instrument"? As well as other terms.